src/common/JSHandle.ts

```diff
@@ -952,10 +956,10 @@ export class ElementHandle<
 
     const { offsetX, offsetY } = await this._getOOPIFOffsets(this._frame);
     const quad = result.model.border;
-    const x = Math.min(quad[0], quad[2], quad[4], quad[6]);
-    const y = Math.min(quad[1], quad[3], quad[5], quad[7]);
-    const width = Math.max(quad[0], quad[2], quad[4], quad[6]) - x;
-    const height = Math.max(quad[1], quad[3], quad[5], quad[7]) - y;
+    const x = Math.min(quad[0]!, quad[2]!, quad[4]!, quad[6]!);
```

Comment:
can we narrow the type by checking the length instead of doing `!`?

Reply:
I think it does not work quite nice for arrays then. Can we change the quad type to be `[number, number x 4]` then? That should help us avoid all those explanations? The quad array is fixed size iirc

Reply:
It is, but the type itself is generated.

Reply:
hm, I see then.